source/whats-new.txt

```diff
@@ -67,6 +67,21 @@ The 5.0 driver release introduces the following features:
 - Changes to the ``ConnectionId`` class. To learn more, see the
   :ref:`Version 5.0 Breaking Changes <java-breaking-changes-v5.0>`.
 
+- A new ``authorizedCollections`` option has been added to the
+  ``listCollections`` command. It can be configured using the
+  ``MongoDatabase.listCollectionNames()`` method. This introduces a
+  :ref:`breaking change to the listCollections
+  
+- The ``listCollections`` command has an ``authorizedCollections`` option. It can be configured using the
+  ``MongoDatabase.listCollectionNames()`` method. This introduces a :ref:`binary
+  breaking change to the listCollections
+  command<java-breaking-changes-v5.0-list-collections>`.
+
+  For more information about ``authorizedCollections``, see the
+  :manual:`listCollections </reference/command/listCollections/>` server manual
+  page. For more information about the ``listCollectionNames()`` method, see the
+  :ref:`Database Collections<java-collection-names>` page.
```

Comment:
These two list items say the same thing, why do they both exist?

Reply:
Removed one. I think there was confusion around the difference between the listCollections command and the listCollectionsNames method.